src/cycles/limited_length.jl

```diff
@@ -0,0 +1,53 @@
+"""
+    simplecycles_limited_length(g, n, ceiling=10^6)
+
+Compute and return at most `ceiling` cycles of length at most `n` of
+the given graph. Both directed and undirected graphs are supported.
+
+### Performance
+The number of cycles grows very fast with the number of vertices and
+the allowed length of the cycles. This function is intended for
+finding short cycles. If you want to find cycles of any length in a
+directed graph, `simplecycles` or `simplecycles_iter` may be more
+efficient.
+"""
+function simplecycles_limited_length(graph::AbstractGraph, n::Integer,
+                                     ceiling = 10^6)
+    cycles = Vector{Int}[]
+    n < 1 && return cycles
+    cycle = Int[]
+    for v = 1:nv(graph)
```

Comment:
This should be `for v in vertices(g)` - we have some optimizations using `Base.OneTo()` here.